From 077f964f3a7ea1476ab47bc4aa3eeb2194965f59 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Tue, 23 Sep 2025 09:25:16 +0200
Subject: [PATCH] dysk: do not lock deps

Version of rust packages specified in Corgo.lock are too old for building.
---
 utils/dysk/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/utils/dysk/Makefile b/utils/dysk/Makefile
index cc559b083..d1743596f 100644
--- a/utils/dysk/Makefile
+++ b/utils/dysk/Makefile
@@ -6,7 +6,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dysk
 PKG_VERSION:=2.8.2
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/Canop/dysk/tar.gz/v$(PKG_VERSION)?
@@ -19,6 +19,8 @@ PKG_LICENSE_FILES:=LICENCE
 PKG_BUILD_DEPENDS:=rust/host
 PKG_BUILD_PARALLEL:=1
 
+RUST_PKG_LOCKED:=0
+
 include $(INCLUDE_DIR)/package.mk
 include ../../lang/rust/rust-package.mk
 
-- 
2.51.0

